### PR TITLE
update scripts for jsc-zen2 to support testing in a container

### PR DESCRIPTION
**Pull request by boegel**
@SebastianAchilles This is a quick port of the equivalent scripts for `generoso`, but it should work.

Only problem is that Singularity or Apptainer are not available yet on `jsc-zen2`, can that be fixed?

### CODE REVIEW

File: eb_from_pr_upload_jsc-zen2.sh [hunk 1]
```diff
@@ -8,15 +8,15 @@
...
-module use $TOPDIR/$USER/Rocky8/zen2/modules/all
+CONTAINER_BIND_PATHS="--bind ${TOPDIR}/$USER --bind ${TOPDIR}/maintainers"
 
 EB_PREFIX=$HOME/easybuild
 export PYTHONPATH=$EB_PREFIX/easybuild-framework:$EB_PREFIX/easybuild-easyblocks:$EB_PREFIX/easybuild-easyconfigs
 # $HOME/.local/bin is added to $PATH for Python packages like archspec installed with 'pip install --user'
 export PATH=$EB_PREFIX/easybuild-framework:$HOME/.local/bin:$PATH
 
-export EASYBUILD_PREFIX=$TOPDIR/$USER/Rocky8/zen2
+export CPU_ARCH=haswell
```
Comment (SebastianAchilles) [inline eb_from_pr_upload_jsc-zen2.sh:18]:
```suggestion
export CPU_ARCH=zen2
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]

**Comment by SebastianAchilles** [comment]
Yes, I will look into making Singularity available on `jsc-zen2`.

File: eb_from_pr_upload_jsc-zen2.sh (hunk 1 above)
Comment (boegel) [reply eb_from_pr_upload_jsc-zen2.sh:18]: Ah, silly copy-pasting mistake 🤦

**Comment by boegel** [comment]
> Yes, I will look into making Singularity available on `jsc-zen2`.

Better go straight for Apptainer?

**Comment by SebastianAchilles** [comment]
This PR can be closed, as `jsc-zen2` is no longer in production and the functionality has been added to `jsc-zen3`.

**Pull request closed without merging**